### PR TITLE
Styling changes

### PR DESCRIPTION
**Pull request by shuyinxu**
-increased margins on text pages
-changed styling for looking for breeder/breed
-other small ui fixes

### CODE REVIEW

**Comment by erikon** [comment]
Hmm, might wanna check out the code climate issue. Something might be wrong with the admin/breeders file?

File: app/models/user.rb [hunk 1]
```diff
@@ -85,7 +85,7 @@ def self.new_token
   end
 
   def admin
-    return true if self.email == "lhsdvm@aol.com"  || self.email == "testuser@berkeley.edu"
+    return true if self.email == "lhsdvm@aol.com"  || self.email == "testuser@berkeley.edu"  || self.email == "xshuyin@berkeley.edu"
```
Comment (erikon) [inline app/models/user.rb]: Should this still be here or was it for testing?

Comment (Raymongdu) [inline app/models/user.rb]: Did you mean to have this here or should this be commented out? In the future might want to make like a development version or something maybe?

Comment (shuyinxu) [reply app/models/user.rb]: that's my bad thanks !

Comment (shuyinxu) [reply app/models/user.rb]: got it!

Comment (Raymongdu) [reply app/models/user.rb]: oh never mind it's already commented on by eric

**Pull request merged**